service/ziti-tunnel/service/state.go

```diff
@@ -595,3 +596,19 @@ func (t *RuntimeState) UpdateNotificationFrequency(notificationFreq int) error {
 
 	return nil
 }
+
+func CleanUpZitiTUNAdapters(tunName string) {
+	log.Info("Invoking ZitiTun adapter cleanup script")
+	tun.WintunPool.DeleteMatchingAdapters(func(wintun *wintun.Adapter) bool {
+		interfaceName, err := wintun.Name()
+		if err != nil {
+			log.Warnf("Could not determine interface name, not removing: %v", err)
+			return false
+		}
+		if strings.HasPrefix(interfaceName, tunName) {
+			log.Infof("Removing old Wintun interface with name : %s", interfaceName)
```

Comment:
I changed this log level to info, because the log level of ziti tun will not be set to trace until this cleanup is done.